kairon/api/app/routers/bot/bot.py

```
@@ -1733,4 +1733,16 @@ async def get_llm_metadata(
     Returns a list of LLMs and their corresponding models available for the bot.
     """
     llm_models = LLMProcessor.fetch_llm_metadata(current_user.get_bot())
-    return Response(data=llm_models)
\ No newline at end of file
+    return Response(data=llm_models)
+
+
+@router.get("/slots/{slot_name}", response_model=Response)
+async def get_slot_actions(
+        slot_name: str = Path(description="slot name", examples=["audio", "order"]),
+        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=TESTER_ACCESS)) -> Response:
```

Ruff

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)